src/App/services/Collisions/Collisions.ts

```diff
@@ -169,21 +138,20 @@ export class Collisions implements IService {
     * What is checked is if "circle" collides with any of the "shots", if so the return which "shot"
     * "circle" collided with.
     */
-   private calcCollisions = (
-      params: { doesThis: PosAndRadiusAndId, collideWithThese: PosAndRadiusAndId[] }
-   ): TCalcCollisionsResult => {
-      const { doesThis, collideWithThese } = params;
+   private calcCollision = (
+      params: { doesThis: PosAndRadiusAndId, collideWithThis: PosAndRadiusAndId }
+   ): boolean => {
+      const { doesThis, collideWithThis } = params;
       
-      for(const shot of collideWithThese) {
-         // Multiplying minDistance if a hack to cause lower hit "box".
-         const minDistance = doesThis.Radius + shot.Radius * 0.8;
-         const xDist = doesThis.x - shot.x;
-         const yDist = doesThis.y - shot.y;
-         const distance = Math.hypot(xDist, yDist);
-         if(distance <= minDistance) {
-            return { collided: true, collidedWithId: shot.id };
-         }
+      // Multiplying minDistance if a hack to cause lower hit "box".
+      const minDistance = doesThis.Radius + collideWithThis.Radius * 0.8;
```

Comment:
I think it makes more sense to do something else than multiplying the 2nd object by 0.8, like either using no multiplication at all or using some fixed number.

Reply:
or using an attribute like "collisionRadiusMultiplier".